extrafiles/tooling/nktooling/__init__.py

```diff
@@ -109,6 +109,12 @@ def getCXXFiles(includeSources = True, includeHeaders = True):
 
 	return files
 
+def getNewCXXFiles():
+	""" Returns a list of all new C++ files added to the repository."""
+	out = subprocess.Popen(("git", "diff", "--name-status", "master"), stdout=subprocess.PIPE)
+	out = subprocess.Popen(("grep", "^A.*\.[ch]pp$"), stdin=out.stdout, stdout=subprocess.PIPE)
+	return subprocess.check_output(("awk", "{ print $2 }"), stdin=out.stdout).decode("utf8").split("\n")[:-1]
```

Comment:
Wouldn't it be cleaner and possibly less error-prone to do the post-processing in Python? Further, `pygit2` could be used to directly use the git repository in Python, but that's minor nit-picking. I'm not saying that this must be changed, after all this is just a helper for CI where we are in a relatively controlled environment.